hist/hist/inc/TH1.h

```diff
@@ -96,6 +98,7 @@ class TH1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
     TList        *fFunctions;       ///<->Pointer to list of functions (fits and user)
     Int_t         fBufferSize;      ///< fBuffer size
     Double_t     *fBuffer;          ///<[fBufferSize] entry buffer
+    TStatistic fXstat;              ///< Statistic about X in fBuffer (max, min, mean, rms, ...)
```

Comment:
This increases the size of all histogram by 5 doubles.  Isn't any of the field redudant (i.e. is there other field than can be dropped)?

Reply:
Yes, this can be done differently, with at most two additonial doubles.